Autocoders/Python/src/fprime_ac/generators/visitors/PortHVisitor.py

```diff
@@ -114,18 +114,17 @@ def _get_args_string(self, obj):
             #
             # Add modifier here - if any...
             if arg.get_modifier() == "pointer":
-                t = t + " *"
+                t += " *"
             elif arg.get_modifier() == "reference":
-                t = t + " &"
+                t += " &"
             elif arg.get_modifier() == "value":
-                t = t + " "
+                t += " "
             elif TypesList.isPrimitiveType(t) or isEnum:
-                t = t + " "
+                t += " "
             else:
                 t = "const " + t + " &"
 
-            arg_str += "{}{}".format(t, arg.get_name())
-            arg_str += ", "
+            arg_str += f"{t}{arg.get_name()}, "
         arg_str = arg_str.strip(", ")
```

Comment:
You can return directly here, as you have done on other lines, right?

Reply:
that's right